packages/math-input/src/components/keypad/index.tsx

```diff
@@ -78,7 +78,14 @@ export default function Keypad(props: Props) {
                 style={styles.tabbar}
             />
 
-            <View style={styles.grid}>
+            <View
+                style={styles.grid}
+                role="grid"
+                tabIndex="0"
+                aria-label="Keypad"
+                aria-rowcount="4"
+                aria-colcount="5"
```

Comment:
**thought**: We're getting more and more details of the rows and columns spread around more places in this keypad (here, in styles, in the `coord` prop).  Wondering if we think this is a liability, and if so, how we'd manage that. 🤔

Reply:
This is a great callout as now there's some pages that overflow. Some pages use 5 columns with a 1 column gap between the sidebar, and some use 6 with no gap. Some don't use all the rows either. Maybe it makes sense to skip these two particular aria tags as it changes so dynamically. 

It would be good to discuss how we want to keep track of rows and columns however!